hoomd/operation.py

```diff
@@ -301,43 +300,18 @@ class see its documentation.
     .. _file: https://github.com/glotzerlab/hoomd-blue/blob/master/ \
         ARCHITECTURE.md
     """
-    pass
-
 
-class _TriggeredOperation(Operation):
-    """Light wrapper around `Operation` for some C++ implementations."""
-    _cpp_list_name = None
 
-    _override_setattr = {'trigger'}
+class TriggeredOperation(Operation):
+    """Operations that include a trigger to determine when to run."""
```

Comment:
Since this class is a part of the docs now, let's add the same note the other classes like `Updater` have about it not being instantiated by users.